MVP/defaults-build.yaml

```diff
@@ -0,0 +1,74 @@
+- defaults:
+    name: defaults-build
+    description: |
+        Managed by Jenkins Job Builder. Do not edit via web.
+    node: 'jslave-platform-rhel7' #[MVP] please replace this value with jenkins slave name/label of your own.
+    #concurrent: true
+    parameters:
+        - string:
+            name: id
+        - string:
+            name: task_id
+        - string:
+            name: CI_TYPE
+        - string:
+            name: CI_MESSAGE
+        - string:
+            name: tag 
+        - string:
+            name: name
+        - string:
+            name: version
+        - string:
+            name: release
```

Comment:
(related to the tag thing above)
It's interesting that there is only "tag" but never "target" from the CI messages that triggered kernel tests. Guess because we use tag in jms-selector.
Will keep both tag/target in the template so it works in both cases.